.github/workflows/add-remove-labels.yml

```diff
@@ -3,7 +3,8 @@ on:
   pull_request_target:
     types: [synchronize]
 
-  pull_request_review:
+  pull_request_review_comment:
+    types: [created]
```

Comment:
we also need to address it when coming from `issue_comment`; needs to be done in the python code

Reply:
`issue_comment` is handled by #L9, isn't this breaking only when leaving a comment on the diff review?

Reply:
missed that this is another workflow